<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminPanelForm</class>
 <widget class="QWidget" name="adminPanelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>adminPanelForm</string>
  </property>
  <widget class="QLabel" name="label_levelDisplay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="rightChangeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поменять
привилегию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileEditButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Заполнить файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileCreateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сделать новый
файл</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Разлогиниться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readFileButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Прочитать файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
